src/pages/components/CertificateForm.js

```diff
@@ -1,16 +1,61 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
 
 import { Grid } from '@digicatapult/ui-component-library'
 
+import CertificateInputFields from './CertificateInputFields'
+import CertificateFormHeader from './CertificateFormHeader'
+import CertificateActionsButtons from './CertificateActionsButtons'
+
 export default function CertificateForm() {
+  const [sdVal, setSdVal] = useState('2024-01-01')
+  const [stVal, setStVal] = useState('00:00')
+  const [edVal, setEdVal] = useState('2024-01-01')
+  const [etVal, setEtVal] = useState('23:55')
+  const [enVal, setEnVal] = useState('')
+  const [szVal, setSzVal] = useState('')
+
+  const handleSdChgeVal = (e) => setSdVal(e.target.value)
+  const handleStChgeVal = (e) => setStVal(e.target.value)
+  const handleEdChgeVal = (e) => setEdVal(e.target.value)
+  const handleEtChgeVal = (e) => setEtVal(e.target.value)
+  const handleEnChgeVal = (e) => setEnVal(e.target.value)
+  const handleSzChgeVal = (e) => setSzVal(e.target.value)
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    const obj = {
+      startDateVal: sdVal,
+      startTimeVal: stVal,
+      endDateVal: edVal,
+      endTimeVal: etVal,
+      energyVal: enVal,
+      sizeVal: szVal,
+    }
+    alert('DATA:\n' + JSON.stringify(obj, null, 2)) // window.location.href = 'https://www.google.com'
+  }
+
   return (
-    <Form>
+    <Form action="" onSubmit={handleSubmit}>
       <Timeline area="timeline">timeline</Timeline>
```

Comment:
can we move `Timeline` out of the `Form` please? It can stay in this component just wrap it and the form in a fragment

Reply:
Done.